tasks/Task.hpp

```diff
@@ -22,25 +31,33 @@ tasks/Task.cpp, and will be put in the lidar_ouster namespace.
          task('custom_task_name','lidar_ouster::Task')
      end
      \endverbatim
-     *  It can be dynamically adapted when the deployment is called with a prefix argument.
+     *  It can be dynamically adapted when the deployment is called with a prefix
+argument.
      */
-    class Task : public TaskBase
-    {
-	friend class TaskBase;
-    protected:
-
-
+    class Task : public TaskBase {
+        friend class TaskBase;
+
+    private:
+        const size_t m_udp_buf_size = 65536;
+        const std::string m_data_destination = "";
+        std::shared_ptr<ouster::sensor::client> m_handle;
+        ouster::sensor::sensor_info m_metadata;
+        std::string m_sensor_hostname;
+        std::unique_ptr<ouster::ScanBatcher> m_scan_batcher;
+        std::unique_ptr<ouster::sensor::packet_format> m_packet_format;
+        double m_vertical_fov = 22.5;
```

Comment:
When you move the vertical_fov into the struct you can add the default value there.

Reply:
Or if we are not going to change it you can leave is as hardcoded VERTICAL_FOV = 22.5 instead of a configurable variable

Reply:
But it isn't a lidar configuration, it is a parameter.

Reply:
@jhonasiv suggested to put it as a property because if we can still use this component if we use another ouster lidar with the same SDK

Reply:
IMO, it's a variable that would vary from other Ouster LIDARs, so it makes sense for it to be a property. You could set the default value on the orogen if you feel like it will most likely remain the same, or you could add a comment suggesting this value on the property

Reply:
Yep, I agree. But since we are receiving it as a property, we don't need to set the default here.